.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -460,6 +463,16 @@ def write(x=''):
             }
         }
 
+        if args.setup_for_sample_courses:
+            worker_dict['primary']['capabilities'].extend([
+                'cpp',
+                'python',
+                'matlab',
+                'pdf-word-count',
+                'et-cetera',
```

Comment:
This label looks like it's applied to gradeables that just use built-in Submitty capabilities? Do we need this as an example? Or could it be made more explicit as to why an sysadmin might want this as a capability tag?